ISSUE_LIFECYCLE.md

```diff
@@ -0,0 +1,37 @@
+# Issue Lifecycle
+
+To ensure a balance between work carried out by the NGINX engineering team while encouraging community involvement on this project, we use the following issue lifecycle. (Note: The issue *creator* refers to the community member that created the issue. The issue *owner* refers to the NGINX team member that is responsible for managing the issue lifecycle.)
+
+1. New issue created by community member.
+
+
+2. Assign issue owner: All new issues are assigned an owner on the NGINX engineering team. This owner shepherds the issue through the subsequent stages in the issue lifecycle.
+
+
+3. Determine issue type: This is done with automation where possible, and manually by the owner where necessary. The associated label is applied to the issue.
+   #### Possible Issue Types
+   `needs more info`: The owner should use the issue to request information from the creator. If we don't receive the needed information within 7 days, automation closes the issue.
+
+   `bug`: The implementation of a feature is not correct.
+
+   `proposal`: Request for a change. This can be a new feature, tackling technical debt, documentation changes, or improving existing features.
```

Comment:
there is a bit of contradiction here.  It starts with "Request for a change." But then it says "new feature".
```suggestion
   `proposal`:  A new feature, tackling technical debt, documentation changes, or improving existing features.
```